lib/src/lifecycle_module.dart

```diff
@@ -127,6 +134,35 @@ abstract class LifecycleModule extends SimpleModule with Disposable {
     }.forEach(_logLifecycleEvents);
   }
 
+  /// If this module is in a transition state, this is the Span capturing the
+  /// transition state.
```

Comment:
This doc comment could use some example usage.

Reply:
I added an example but I'm not sure if it's what you meant.